google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceInfoTest.java

```diff
@@ -64,6 +69,8 @@ public void testBuildInstanceInfo() {
     assertThat(info.getProcessingUnits()).isEqualTo(2000);
     assertThat(info.getState()).isEqualTo(InstanceInfo.State.READY);
     assertThat(info.getLabels()).containsExactly("env", "prod", "region", "us");
+    assertThat(info.getUpdateTime()).isEqualTo(Timestamp.ofTimeMicroseconds(36000));
```

Comment:
nit: (here and elsewhere) the prefered method for testing equality is `assertEquals(expected, actual)`, even if that breaks with what is already in a specific method (even better is to change the existing `assertThat(..).isEqualTo(..)` to `assertEquals(..)` in the process).